views/resources.sql

```diff
@@ -0,0 +1,28 @@
+DROP VIEW IF EXISTS aws_resources;
+
+do $$
+declare
+  tbl text;
+  strSQL text = '';
+begin
+ -- iterate over every table in our information_schema that has an `arn` column available
+FOR tbl IN SELECT table_name from information_schema.columns where table_name like 'aws_%s' and COLUMN_NAME = 'account_id'
+           intersect
+select table_name from information_schema.columns where table_name like 'aws_%s' and COLUMN_NAME = 'arn'
+ LOOP 
+     -- UNION each table query to create one view
+ 	 IF NOT (strSQL = ''::text) THEN
+	      strSQL = strSQL || ' UNION ALL ';
+	 END IF;
+	 -- create an SQL query to select from table and transform it into our resources view schema
+	 strSQL = strSQL || format('select  cq_id,  cq_meta, %L as cq_table, account_id, %s as region, arn,
+		  					    COALESCE(%s, (cq_meta->>''last_updated'')::timestamp) as fetch_date
+							   FROM %s', tbl,
```

Comment:
nit: I would still add tags here, as it allows for many useful queries, the breaking up of the ARN might be a little tedious to understand.

Reply:
done